Mapper/BaseGroupedMapper.php

```diff
@@ -257,6 +263,16 @@ public function hasOpenTab()
     abstract protected function setTabs(array $tabs);
 
     /**
+     * TODO: make this method abstract before releasing 4.0
+     *
+     * @return string
+     */
+    /*abstract*/ protected function getName()
```

Comment:
This will make CS fixer crying.

Please follow this adopted format: https://github.com/sonata-project/dev-kit/pull/78